docker-compose.yml

```diff
@@ -37,6 +37,11 @@ services:
       retries: 5
       timeout: 10s
 
+  dqEdit:
+    image: ghcr.io/dqualizer/dqedit-react:react-dev
```

Comment:
As agreed today, let's aim at a stable release version before officially releasing the new showacse version.